### ISSUE TITLE
Wrong justify direction in RTL

### ISSUE THREAD

**Issue by alifaraji** [WordPress/gutenberg#41370]
### Description

There are two flex divs in post status panel assigned with left values instead of right, Its causing UI problems with RTL versions.
in `wp-includes/css/dist/edit-post/style-rtl.css`
changing
```
.edit-post-post-visibility {
  justify-content: left;
}

.edit-post-post-schedule {
  justify-content: left;
}
```
To
```
.edit-post-post-visibility {
  justify-content: right;
}

.edit-post-post-schedule {
  justify-content: right;
}
```
Seems to fix the problem.

### Step-by-step reproduction instructions

1. Go to Post Editor

### Screenshots, screen recording, code snippet

![LTR verstion](https://wpvar.com/img/trac/1/ltr.png)
![RTL verstion](https://wpvar.com/img/trac/1/rtl.png)

### Environment info

- WordPress 6.0

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by alifaraji** [comment]
https://core.trac.wordpress.org/ticket/55743

**Comment by sabernhardt** [comment]
The `justify-content` property was merged in #23578.

Instead of using `left` and `right`, could the value be `start` (or `flex-start`) for both LTR and RTL stylesheets?

_Note for props: alifaraji is wpvar on WP.org_
